extensions/sql-migration/src/constants/strings.ts

```diff
@@ -425,6 +425,8 @@ export const DATA_SOURCE_CONFIGURATION_PAGE_TITLE = localize('sql.migration.data
 export const DATABASE_BACKUP_PAGE_DESCRIPTION = localize('sql.migration.database.page.description', "Select the location of the database backups to use during migration.");
 export const DATABASE_BACKUP_NC_NETWORK_SHARE_RADIO_LABEL = localize('sql.migration.nc.network.share.radio.label', "My database backups are on a network share");
 export const DATABASE_BACKUP_NC_BLOB_STORAGE_RADIO_LABEL = localize('sql.migration.nc.blob.storage.radio.label', "My database backups are in an Azure Storage Blob Container");
+export const DATABASE_BACKUP_SQL_VM_PAGE_BLOB_INFO = localize('sql.migration.sql.vm.page.blob.info', "For source servers running SQL Server 2014 or below, you must store your database backups in an Azure Storage Blob Container instead of uploading them using the network share option. Additionally, you must store the backup files as page blobs, as backups to block blobs are supported only in SQL Server 2016 or later. Learn more: {0}");
```

Comment:
```suggestion
export const DATABASE_BACKUP_SQL_VM_PAGE_BLOB_INFO = localize('sql.migration.sql.vm.page.blob.info', "For source servers running SQL Server 2014 or below, you must store your database backups in an Azure Storage Blob Container instead of uploading them using the network share option. ");
```

can you take that part out? I ended up adding a check to see the blob type, and so we no longer this warning. If they incorrect blob type, I block them with instructions

Reply:
see this PR for details: https://github.com/microsoft/azuredatastudio/pull/21567